docs/source/cuda._sanitizer.rst

```diff
@@ -0,0 +1,95 @@
+.. currentmodule:: torch.cuda._sanitizer
+
+torch.cuda._sanitizer
+=====================
+
```

Comment:
I think it would be good to add a box/warning at the very top that this is a prototype feature, with some explanation of what it means in terms of support, of stability, ...

It would also be nice, later on, to have a kind of list of known feature gaps, i.e., things that CSAN doesn't yet support.